packages/react/CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # @lens-protocol/react
 
+## 2.0.0-alpha.23
+
+### Minor Changes
+
+- 493895b8: **feat:** self-funded `useFollow`
+- fdd0073d: **feat:** support for self-funded `useCreateQuote`
+- 5d243a83: **feat:** support for self-funded `useCreateMirror`
+- b37f6f4e: **feat:** self-funded `useSetProfileMetadata`
+- 2e351a8c: fixes called property of hooks to update correctly when hook returns void
+- 2698fc65: **feat:** support for self-funded `useCreatePost`
+- 6a25dc02: **feat:** support self-funded `useOpenAction`
+- 40abddd9: **feat:** support for self-funded `useCreateComment`
+
+### Patch Changes
```

Comment:
as it's still alpha, the split for minor/patch doesn't really make much sense and it's easier to read if all points were in a single list, could we merge them (just in the changelog, better to not touch changeset files at this point).